pkg/cfn/manager/create_tasks.go

```diff
@@ -26,9 +27,9 @@ func (c *StackCollection) NewTasksToCreateClusterWithNodeGroups(nodeGroups []*ap
 	)
 
 	appendNodeGroupTasksTo := func(taskTree *tasks.TaskTree) {
-		nodeGroupTasks := c.NewUnmanagedNodeGroupTask(nodeGroups, supportsManagedNodes, false)
-
-		managedNodeGroupTasks := c.NewManagedNodeGroupTask(managedNodeGroups, false)
+		vpcImporter := vpc.NewStackConfigImporter(c.MakeClusterStackName())
```

Comment:
@aclevername this is the silly thing I screwed up. I initially had this as the cluster name, when it needs the stack name 🤦‍♀️

Reply:
nice